src/main/java/ui/RepositorySelector.java

```diff
@@ -49,6 +52,17 @@ private void setupComboBox() {
                 onValueChangeCallback.accept(repoId);
             }
         });
+        
+        // remove focus from the repo selector
+        comboBox.addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+            event.consume();
+            if (event.getCode() == KeyboardShortcuts.REMOVE_FOCUS) {
+                if (ui.isTestMode()) {
```

Comment:
hmm yeah although now that i think about it, could we check if the repo selector itself has focus instead of checking for the event?